src/index.html

```diff
@@ -0,0 +1,109 @@
+      <p class="slide__title">Новая версия популярного микроавтобуса</p>
+      <p class="slide__subtitle">С ЭЛЕКТРО ДВИГАТЕЛЕМ</p>
+      <button class="slide__button" type="button">Подробнее</button>
+      <img class="slide__photo" src="../images/bus.png" alt="Фото автобуса">
+    </section>
+    <section class="pros-cons">
+      <div class="pros-cons__description">
+        <p class="pros-cons__text"><span class="pros-cons__text-bold">VW T1 Electro:</span> Отделение Volkswagen
+          Commercial Vehicles представило концепт
+          e&#8209;Bulli: классический вэн T1 1966 года
+          с&nbsp;электрической силовой установкой.
+          По&nbsp;словам производителя, проект получит коммерческое продолжение, и через фирму&#8209;партнера eClassics
+          наборы для
+          конверсии микроавтобусов в электрокары
+          и&nbsp;уже готовые машины будут продавать частным клиентам.
+        </p>
+      </div>
+      <dl class="pros-cons__container">
+        <div class="pros-cons__wrapper pros-cons__advantages">
+          <dt class="pros-cons__title pros-cons__advantages-title">ПЛЮСЫ</dt>
+          <dd class="pros-cons__item pros-cons__advantages-item">+ Экологичный</dd>
+          <dd class="pros-cons__item pros-cons__advantages-item">+ Экономный</dd>
+          <dd class="pros-cons__item pros-cons__advantages-item">+ Тихий</dd>
+        </div>
+        <div class="pros-cons__wrapper pros-cons__disadvantages">
+          <dt class="pros-cons__title pros-cons__disadvantages-title">МИНУСЫ</dt>
+          <dd class="pros-cons__item pros-cons__disadvantages-item">- Дорогой</dd>
+          <dd class="pros-cons__item pros-cons__disadvantages-item">- Подзарядка</dd>
+        </div>
+      </dl>
+    </section>
+    <section class="gallery">
+      <div class="gallery__slider-content">
+        <img class="gallery__photo" src="../images/bus-gallery.png" alt="Фото автобуса">
+        <div class="gallery__slider">
+          <h2 class="section-title gallery__slider-title">Галерея</h2>
+          <div class="gallery__wrapper">
+            <button type="button" class="gallery__slider-controls gallery__slider-controls_dir_left active-el"></button>
+            <button type="button"
+              class="gallery__slider-controls gallery__slider-controls_dir_right active-el"></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="press">
+      <h2 class="section-title press__title">Пресса</h2>
+      <div class="press__news">
+        <p class="press__news-normal"><span class="press__news-bold">Engadget:</span> VW’s e-BULLI concept shows how
+          your classic van can become an EV.</p>
+        <div class="press__container">
+          <div class="press__wrapper">
+            <button type="button" class="press__circle-button press__circle-button_active active-el"></button>
+            <button type="button" class="press__circle-button active-el"></button>
+          </div>
```

Comment:
Здесь кнопки лучше сделать списком:
```suggestion
          <ul class="press__wrapper">
            <li><button type="button" class="press__circle-button press__circle-button_active active-el"></button></li>
            <li><button type="button" class="press__circle-button active-el"></button></li>
          </ul>
```